<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QWidget" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pluginsTab">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="differentApiButton">
           <property name="toolTip">
            <string>Will reload all plugins</string>
           </property>
           <property name="text">
            <string>Accept different API Versions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="pluginsList"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="reloadButton">
             <property name="text">
              <string>Reload all</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="settingsButton">
             <property name="text">
              <string>Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="createButton">
             <property name="text">
              <string>Create new</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="repositoryButton">
             <property name="text">
              <string>Get more</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="7" column="1">
          <widget class="QLineEdit" name="apiEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&amp;Keyword:</string>
           </property>
           <property name="buddy">
            <cstring>keywordEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;Name:</string>
           </property>
           <property name="buddy">
            <cstring>nameEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;Version:</string>
           </property>
           <property name="buddy">
            <cstring>versionEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>API &amp;Version:</string>
           </property>
           <property name="buddy">
            <cstring>apiEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&amp;Description:</string>
           </property>
           <property name="buddy">
            <cstring>descriptionEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Re&amp;quired:</string>
           </property>
           <property name="buddy">
            <cstring>requiredApiEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>A&amp;uthor:</string>
           </property>
           <property name="buddy">
            <cstring>authorEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QLineEdit" name="requiredApiEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="nameEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLineEdit" name="versionEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="3">
          <widget class="QLineEdit" name="authorEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="QTextEdit" name="descriptionEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="3">
          <widget class="QLineEdit" name="keywordEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="consoleTab">
      <attribute name="title">
       <string>Scripting Console</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="colorsBox">
         <property name="title">
          <string>Colors</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Bac&amp;kground:</string>
            </property>
            <property name="buddy">
             <cstring>backgroundColorButton</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="backgroundColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Te&amp;xt:</string>
            </property>
            <property name="buddy">
             <cstring>textColorButton</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="textColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="fontBox">
         <property name="title">
          <string>Font</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Famil&amp;y:</string>
            </property>
            <property name="buddy">
             <cstring>fontFamilyCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFontComboBox" name="fontFamilyCombo"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Si&amp;ze:</string>
            </property>
            <property name="buddy">
             <cstring>fontSizeSpin</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="fontSizeSpin"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="tabsBox">
         <property name="title">
          <string>Tabs</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="tabcompleteButton">
            <property name="text">
             <string>Tabcomplete</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="spacesButton">
              <property name="text">
               <string>Insert spaces</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="tabwidthLabel">
              <property name="text">
               <string>Tabwi&amp;dth:</string>
              </property>
              <property name="buddy">
               <cstring>tabwidthSpin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="tabwidthSpin"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="startupBox">
         <property name="title">
          <string>Startup</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Run script:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="scriptEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="scriptButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="silentButton">
            <property name="text">
             <string>Silent</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
